Source/WebKit/WebProcess/GPU/graphics/RemoteImageBufferProxy.h

```diff
@@ -116,11 +116,30 @@ class RemoteImageBufferProxy : public WebCore::ConcreteImageBuffer<BackendType>
         m_receivedFlushIdentifierChangedCondition.notifyAll();
     }
 
-    void setNeedsFlush(bool needsFlush) final
+    void setNeedsFlush(bool needsFlush)
     {
         m_needsFlush = needsFlush;
     }
```

Comment:
I think this function can be deleted since it is now used by RemoteImageBufferProxy only. The callers can just set the member `m_needsFlush` directly.